Source/Particles/ERFPC.H

```diff
@@ -215,12 +216,17 @@ class ERFPC : public amrex::ParticleContainer<  ERFParticlesRealIdxAoS::ncomps,
         void initializeParticlesDefaultHydro (const std::unique_ptr<amrex::MultiFab>& a_ptr = nullptr);
         /*! Default particle initialization */
         void initializeParticlesUniformDistribution (const std::unique_ptr<amrex::MultiFab>& a_ptr = nullptr);
+        void initializeParticlesUniformDistributionInBox (const std::unique_ptr<amrex::MultiFab>& a_ptr,
+                                                          const amrex::RealBox& particle_box);
 
     protected:
 
         bool m_advect_w_flow;               /*!< advect with flow velocity */
         bool m_advect_w_gravity;            /*!< advect under gravitational force */
 
+        amrex::RealBox particle_box;
```

Comment:
Can you rename this to `init_particle_box` or `m_particle_box`? I imagine `particle_box` will get used as a local variable name at various places.